### PR TITLE
chore: README patch

### PR DESCRIPTION
**Pull request by Murtatrxx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,19 +15,16 @@
...
-* Hybrid commands
-* Lightweight and customizable
-* ESM, CommonJS and TypeScript support
-* A powerful CLI and awesome community-made plugins
+- For you. A framework that's tailored to your exact needs.
+- Lightweight. Does a lot while being small.
+- Latest features. Support for discord.js v14 and all of its interactions.
+- Hybrid, customizable and composable commands. Just how you like.
+- Start quick. Plug and play or customize to your liking.
+- Embraces reactive programming. For consistent and reliable backend.
```
Comment (jacoobes) [inline README.md:23]: Embraces reactive programming. For consistent and reliable backend. -> revert

File: README.md [hunk 2]
```diff
@@ -15,19 +15,16 @@
...
+- Logger. Customize, handle errors and do much more.
+- Active and growing community, always here to help. [Join us](https://sern.dev/discord)
+- Use it with TypeScript, ESM, or CommonJS. All welcome.
```
Comment (jacoobes) [inline README.md:26]: Use it with TypeScript or JavaScript. CommonJs and ESM supported

File: README.md [hunk 3]
```diff
@@ -15,19 +15,16 @@
...
+- Unleash the full potential with a powerful CLI and awesome plugins.
```
Comment (jacoobes) [inline README.md:27]: Unleash its full potential

File: README.md [hunk 4]
```diff
@@ -15,19 +15,16 @@
...
-* Lightweight and customizable
-* ESM, CommonJS and TypeScript support
-* A powerful CLI and awesome community-made plugins
+- For you. A framework that's tailored to your exact needs.
+- Lightweight. Does a lot while being small.
+- Latest features. Support for discord.js v14 and all of its interactions.
+- Hybrid, customizable and composable commands. Just how you like.
+- Start quick. Plug and play or customize to your liking.
+- Embraces reactive programming. For consistent and reliable backend.
+- Logger. Customize, handle errors and do much more.
```
Comment (jacoobes) [inline README.md:24]: Switch and customize how errors are handled, logging, and more

File: README.md [hunk 5]
```diff
@@ -15,19 +15,16 @@
...
-- Customizable logger, error handling, and more
-- Active development and growing [community](https://sern.dev/discord)
-## 👀 Quick Look
-
-* Support for discord.js v14 and all interactions
-* Hybrid commands
-* Lightweight and customizable
-* ESM, CommonJS and TypeScript support
-* A powerful CLI and awesome community-made plugins
+- For you. A framework that's tailored to your exact needs.
```
Comment (jacoobes) [inline README.md:18]: too cliche, remove

File: README.md [hunk 6]
```diff
@@ -15,19 +15,16 @@
...
+- Lightweight. Does a lot while being small.
+- Latest features. Support for discord.js v14 and all of its interactions.
+- Hybrid, customizable and composable commands. Just how you like.
```
Comment (jacoobes) [inline README.md:21]: Create customizable commands, just how you like

File: README.md [hunk 7]
```diff
@@ -15,19 +15,16 @@
...
+- Start quick. Plug and play or customize to your liking.
```
Comment (jacoobes) [inline README.md:22]: Remove

**Review by jacoobes — CHANGES_REQUESTED** [review]

File: README.md (hunk 7 above)
Comment (jacoobes) [reply README.md:22]: line 22

File: README.md (hunk 5 above)
Comment (jacoobes) [reply README.md:18]: line 18

File: README.md (hunk 4 above)
Comment (jacoobes) [reply README.md:24]: line 24

File: README.md (hunk 1 above)
Comment (jacoobes) [reply README.md:23]: revert to original

Comment (Murtatrxx) [reply README.md:23]: What's the difference, i made it to keep the style consistent.

File: README.md (hunk 7 above)
Comment (Murtatrxx) [reply README.md:22]: This was on the line 20 in the old README, are you sure?

Comment (jacoobes) [reply README.md:22]: Change Start quick to Start quickly

**Review by jacoobes — CHANGES_REQUESTED** [review]

**Review by jacoobes — APPROVED** [review]

**Pull request merged**